.github/workflows/ci.yaml

```diff
@@ -56,6 +56,8 @@ jobs:
           node-version-file: ".nvmrc"
       - name: Install Node.js dependencies
         run: npm install
+      - name: Install Playwright browsers
+        run: npx playwright install --with-deps
```

Comment:
create a `scripts/postinstall.ts` script that invokes this command. invoke it from the `postinstall` script on the package.json. For now, simply call this command only when `process.env.CI`